prov/verbs/src/verbs_info.c

```diff
@@ -464,7 +464,7 @@ static inline int fi_ibv_get_qp_cap(struct ibv_context *ctx,
 	return ret;
 }
 
-static size_t fi_ibv_mtu_type_to_len(enum ibv_mtu mtu_type)
+static ssize_t fi_ibv_mtu_type_to_len(enum ibv_mtu mtu_type)
```

Comment:
You can make this an int, since it's assigned to one.  Then this will be one less place the windows build will complain.